python/py-isoduration/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-isoduration
+version             20.11.0
+revision            0
+categories-append   devel
+supported_archs     noarch
+
+python.versions     38 39 310 311 312
+python.pep517       yes
+
```

Comment:
you will need to specify a `license`

Reply:
There's no license listed in the source package and macports guide does not list an option for "missing" or "unknown" license.

Reply:
upstream lists a license both on the PyPI package as well as GitHub: https://github.com/bolsote/isoduration/blob/master/LICENSE

Reply:
I mis-worded my response. The license seems to be a "custom" license, not an "official" one with a proper name. So the name of the license is missing. Copy-pasting the whole license does not seem reasonable.

Reply:
`ISC` is a perfectly fine license, there are many other ports in MacPorts that use the same

Reply:
Does ISC stand for some kind of a freeform license? Should it be always, when other licenses are not applicable, used?

Reply:
no, there is nothing "freeform" about it (it's specified in [SPDX](https://spdx.org/licenses/ISC.html)) and it shouldn't always be used. It should be used when upstream specifies it as their license.

Reply:
Right, now I see, pypi lists it indeed as ISC license. The license file in the source package has no mention of the license name.